packages/salesforcedx-vscode-core/src/commands/deleteSource.ts

```diff
@@ -113,15 +114,15 @@ export async function deleteSource(sourceUri: vscode.Uri) {
       const errorMessage = nls.localize(
         'delete_source_select_file_or_directory'
       );
-      telemetryService.sendException('force_source_delete', errorMessage);
+      telemetryService.sendException('delete_source', errorMessage);
```

Comment:
Either change this to `project_delete_source` or change Line 42 to `delete_source` the two log names don't match.